### PR TITLE
az*: update outdated links

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by ikks — APPROVED** [review]
Spanish translations and links are GTG.

**Review by renie — APPROVED** [review]
pt_BR translation LGTM

File: pages.fr/common/az.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 > La CLI officielle de Microsoft Azure.
 > Certaines commandes comme `az login` ont leur propre documentation.
-> Plus d'informations : <https://learn.microsoft.com/cli/azure>.
+> Plus d'informations : <https://learn.microsoft.com/cli/azure/reference-index>.
```
Comment (noraj) [inline pages.fr/common/az.md:5]: Question for other reviewers: should the doc links stay the same accross all translations, or should they be localized when a translation exist?
Here the french translation is: https://learn.microsoft.com/fr-fr/cli/azure/reference-index

Comment (noraj) [reply pages.fr/common/az.md:5]: Thanks.

**Comment by noraj** [comment]
> @noraj can you fix the outdated French translation?

✅

**Review by kant — APPROVED** [review]
Spanish chapter: LGTM

**Review by karthik-script — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Review by Zamoca42 — APPROVED** [review]

**Review by gutjuri — APPROVED** [review]

**Review by spageektti — APPROVED** [review]

**Comment by renie** [comment]
IMO we could wait for @mebeim , @tansiret or @Magrid0 for 24 more hours. If we get no interaction we could merge it already.

**Review by tansiret — APPROVED** [review]
LGTM

**Pull request merged**